src/lender/admin/pool.sol

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2020 Centrifuge
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.15 <0.6.0;
+
+import "tinlake-auth/auth.sol";
+
+interface AssessorLike {
+    function file(bytes32 name, uint256 value) external;
+}
+
+interface LendingAdapter {
+    function raise(uint256 amount) external;
+    function sink(uint256 amount) external;
+}
+
+contract MemberlistLike {
+    function updateMember(address usr, uint256 validUntil) public;
+    function updateMembers(address[] memory users, uint256 validUntil) public;
+}
+
+// Wrapper contract for various pool management tasks.
+contract PoolAdmin is Auth {
+  
```

Comment:
I'd add events similarly to how we use them in: https://github.com/centrifuge/tinlake-maker-lib/blob/master/src/mgr.sol#L87 (especially for file)